server.js

```diff
@@ -1,5 +1,23 @@
-const app = require('./app')
+const app = require("./app");
+const mongoose = require("mongoose");
+require("dotenv").config();
 
-app.listen(3000, () => {
-  console.log("Server running. Use our API on port: 3000")
-})
+const PORT = process.env.PORT || 3000;
+const uriDb = process.env.DB_HOST;
+
+const connection = mongoose.connect(uriDb, {
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
+});
+
+connection
+  .then(() => {
+    console.log(`Database connection successful`);
+    app.listen(PORT, function () {
```

Comment:
```suggestion
    app.listen(PORT,  () => {
```
Trzymaj się jednego sposobu pisania w miarę możliwości